src/common/Surface.js

```diff
@@ -264,7 +264,9 @@
             ;
         }
 
-        this._menu.element().node().parentNode.appendChild(this._menu.element().node()); // Make sure menu is on top (Z-Order POV)
+        if (this._menu.element() && this._menu.element().node() && this._menu.element().node().parentNode) {
```

Comment:
this is to just check th menu exists and is a child before doing the Z order fix?